curriculum/challenges/english/07-scientific-computing-with-python/learn-special-methods-by-building-a-vector-space/65faaed8f7a9772f023ea816.md

```diff
@@ -0,0 +1,75 @@
+---
+id: 65faaed8f7a9772f023ea816
+title: Step 48
+challengeType: 20
+dashedName: step-48
+---
+
+# --description--
+
+Vectors can be multiplied by a *scalar*, i.e. a number that multiplies each single components. The result of a scalar multiplication is a vector with the same orientation of the original vector but different magnitude.
+
+Create an `if` statement to check if `other` is an `int` or a `float`. Use the `isinstance` function and pass it the tuple `(int, float)` as the second argument.
```

Comment:
It would be good to specify that this condition is for scalar multiplication, the dot/cross product would be catered in the other clauses